koku/masu/management/commands/migrate_trino_tables.py

```diff
@@ -224,26 +237,38 @@ def drop_tables(tables, schema):
 
 def add_columns_to_tables(list_of_cols: ListAddColumns, schema: str):
     """add specified columns with datatypes to the tables"""
+    table_exists_cache = {}
     for col in list_of_cols.list:
-        LOG.info(f"adding column {col.column} of type {col.datatype} to table {col.table}")
-        sql = f"ALTER TABLE {col.table} ADD COLUMN IF NOT EXISTS {col.column} {col.datatype}"
-        try:
-            result = run_trino_sql(sql, schema)
-            LOG.info(f"ALTER TABLE result: {result}")
```

Comment:
Looking at the [ALTER TABLE docs](https://trino.io/docs/current/sql/alter-table.html), we could just update this sql statement to be:
```
f"ALTER TABLE IF EXISTS {col.table} ADD COLUMN IF NOT EXISTS {col.column} {col.datatype}"
```